### PR TITLE
Reduce bad data

### PR DESCRIPTION
**Pull request by Omicron7**
This should reduce the number of bad events and users we are stitching. This is implemented in the batch side, but needed to also happen in realtime pipeline.

### CODE REVIEW

File: models/identity-stitcher.js [hunk 1]
```diff
@@ -20,6 +20,14 @@ const {
   without,
   zipObject
 } = require('lodash')
+const TestUserGuids = [
+  '49e1f2f9-55cc-6c10-58ff-b9b46ca79579' // test@test.com
```
Comment (wrandall22) [inline models/identity-stitcher.js]: You can add `fd8c0e67-d19e-49c5-94c2-ff737302d2bc` to this list.

File: models/identity-stitcher.js [hunk 2]
```diff
@@ -20,6 +20,14 @@ const {
...
+]
+const TestUserGRIds = [
+  '6017a717-251c-434f-aa2f-e4279328fa59', // test@test.com
+  'f19c4ec2-057d-4b7b-958b-e88452881a28',
+  '2c4b60d4-7b21-43f7-836f-4f255330ecd2'
```
Comment (wrandall22) [inline models/identity-stitcher.js]: You can add `8a9f5933-b4e1-4ec2-a072-f5ff34e349e6` to this list.

File: models/identity-stitcher.test.js [hunk 3]
```diff
@@ -26,6 +27,18 @@ describe('IdentityStitcher', () => {
     return expect(IdentityStitcher(new Event())).rejects.toThrowError(UnknownUserError)
   })
 
+  it('event with know test user identities (sso_guid) should throw KnownTestUserError', () => {
+    user.sso_guid = ['49e1f2f9-55cc-6c10-58ff-b9b46ca79579']
+    jest.spyOn(User, 'fromEvent').mockImplementation(() => user)
+    return expect(IdentityStitcher(new Event())).rejects.toThrowError(KnownTestUserError)
+  })
+
+  it('event with know test user identities (gr_master_person_id) should throw KnownTestUserError', () => {
```
Comment (wrandall22) [inline models/identity-stitcher.test.js]: In both of these `it` statements, `know` should be `known`.

File: models/identity-stitcher.js [hunk 4]
```diff
@@ -21,12 +21,14 @@ const {
   zipObject
 } = require('lodash')
 const TestUserGuids = [
-  '49e1f2f9-55cc-6c10-58ff-b9b46ca79579' // test@test.com
+  '49e1f2f9-55cc-6c10-58ff-b9b46ca79579', // test@test.com
+  'fd8c0e67-d19e-49c5-94c2-ff737302d2bc' // bill.test@tester.com
```
Comment (wrandall22) [inline models/identity-stitcher.js]: Should actually be `bill.test.tester@stage.com`

Comment (Omicron7) [reply models/identity-stitcher.js]: Both emails are associated with GR entity, I picked one.

**Pull request merged**